<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eaton_Dialog</class>
 <widget class="QDialog" name="eaton_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PP Eaton</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Wells</string>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Prediction</string>
      </property>
      <widget class="QPushButton" name="predict_Button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Predict</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="n_lineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string> N:</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Velocity:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="velocity_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="well_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="obp_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>OBP:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="nct_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NCT:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="n_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>141</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Horizons</string>
      </property>
      <widget class="QPushButton" name="plot_horizon_Button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QListWidget" name="horizon_listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>171</height>
        </rect>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Save</string>
      </property>
      <widget class="QPushButton" name="save_Button">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icon/save_icon</normaloff>:/icon/save_icon</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>_eaton</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
